client/blocks/inline-help/index.jsx

```diff
@@ -145,6 +170,11 @@ class InlineHelp extends Component {
 
 		return (
 			<div className="inline-help">
+				<WpcomChecklist
+					viewMode="notification"
+					setNotification={ this.setNotification }
+					storedTask={ storedTask }
+				/>
```

Comment:
Can you explain what's going on here? I wonder if we could find a simpler way to do this.

Reply:
Hmm, I might be getting it now:

We want to show the notification dot if there are some tasks left to be completed. But the idea behind `storedTask` is to hide that dot once the user actually clicks on the checklist navigation element -- for some instant UI feedback (so they see they're doing the 'right thing' to make that dot go away) -- even tho this still doesn't mean the checklist has been completed yet. So we're effectively introducing a new criterion here.

An alternative without `storedTask` that comes to mind would be to check whether we're currently on the checklist page, and always hide the notification dot in that case.

Reply:
Here's how we envisioned this notification piece working:

* User with an incomplete checklist sees the orange notification dot and clicks on the help icon.
* A matching dot is shown next to the checklist count in the help menu.
* When the user clicks on the button to navigate to the checklist, both notification dots disappear. This clears the notification until...
* The dots reappear when a new item on the checklist becomes available. This could be because the user completes an item.

We could hide the dot only on the checklist page, but the idea was to hide it everywhere until a task is completed (to avoid it being too annoying -- we want to draw people's attention, but not spam them). 

Let me know if that makes more sense and if there's a better approach to use!

Reply:
That's very helpful, thanks for the clarification!